meta-ibm/meta-witherspoon/recipes-phosphor/fans/phosphor-fan_%.bbappend

```diff
@@ -30,11 +30,23 @@ SYSTEMD_ENVIRONMENT_FILE_phosphor-cooling-type_append_ibm-ac-server = " ${@compo
 SYSTEMD_ENVIRONMENT_FILE_phosphor-cooling-type_append_mihawk = " ${@compose_list(d, 'COOLING_ENV_FMT', 'OBMC_CHASSIS_INSTANCES')}"
 
 #These services are protected by the watchdog
-SYSTEMD_OVERRIDE_phosphor-fan-control_witherspoon += "fan-watchdog-monitor.conf:phosphor-fan-control-init@0.service.d/fan-watchdog-monitor.conf"
```

Comment:
So you removed fan-control from watchdog protection?  Am I missing where you added it back?

Reply:
I removed `phosphor-fan-control-init` service from watchdog protection since that service is no longer installed on witherspoon with `phosphor-fan-control` running at standby and handling setting fans to full speed upon poweron.